### DISCUSSION TITLE
Incluir link para "copiar" no alert

### DISCUSSION THREAD

**Discussion by giovanna-abrantes** [TiendaNube/nimbus-design-system#60] [Q&A] [answered]
Oláa! 
Queria trazer uma sugestão para o componente de Alert. Em Pago Nube havíamos previsto o comportamento abaixo:
<img width="883" alt="Captura de Tela 2022-11-29 às 14 21 54" src="https://user-images.githubusercontent.com/107882097/204599102-0783fe7b-43b4-44f4-829f-7d85d270461a.png">
Mas, em desenvolvimento, percebemos que o componente não permite que o link do alert faça uma ação de "copiar", apenas permite uma ação de redirecionar para outro local. 

Existe alguma diretriz para esse caso? 
Obrigada :)

**Comment by juanchigallego** [comment, marked as answer]
Bom dia Gio, tudo bem?

Na v1 do Nimbus, não é possível fazer isso, porque o componente `Link` não pode receber eventos `onClick` (só o `Button` pode). Além disso, vocês podem usar o `Alert` e `Link` da v2 do Nimbus, onde sim conseguiriam lograr esses comportamentos.

Eu trouxe [aqui um codesandbox](https://codesandbox.io/s/pensive-raman-g8it9k?file=/src/App.tsx) de exemplo de como lograr isso usando os novos componentes. Deixo também o codigo embaixo para lograr o Alert com link de copiar texto 👇🏼 

```jsx
import { Alert } from "@nimbus-ds/alert";
import { Link } from "@nimbus-ds/link";
import { Icon } from "@nimbus-ds/icon";
import { CopyIcon } from "@tiendanube/icons";

export default function App(): JSX.Element {
  const copyText = () => {
    alert("Copied hola@tiendanube.com to clipboard");
  };
  return (
    <div className="App">
      <Alert>
        Por tu seguridad, para alterar estos datos deberás contactar soporte o
        escribir a hola@tiendanube.com
        <Link onClick={copyText}>
          <Icon source={<CopyIcon />} />
          Copiar e-mail
        </Link>
      </Alert>
    </div>
  );
}
```

Espero poder haber ajudado vocês. Cualquer dúvida me avisan!
